tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -1,50 +1,89 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 """Contractors based on `opt_einsum`'s path algorithms."""
 
 import functools
 import opt_einsum
-from typing import Any, Callable, Dict, Optional, List, Set
+from typing import Any, Callable, Dict, Optional, List, Set, Sequence
 from tensornetwork import network
+from tensornetwork import network_components
 from tensornetwork.contractors.opt_einsum_paths import utils
 
 
 def base(net: network.TensorNetwork,
-         algorithm: Callable[[List[Set[int]], Set[int], Dict[int, int]],
-                             List]) -> network.TensorNetwork:
-  """Base method for all `opt_einsum` contractors.
+         algorithm: Callable[[List[Set[int]], Set[int], Dict[int, int]], List],
+         output_edge_order: Sequence[network_components.Edge] = None
+        ) -> network.TensorNetwork:
+    """Base method for all `opt_einsum` contractors.
 
   Args:
     net: a TensorNetwork object. Should be connected.
     algorithm: `opt_einsum` contraction method to use.
-
+    output_edge_order: An optional list of edges. Edges of the 
```

Comment:
Remove the `optional`.

Can you also reformat this to just be one tab in rather than aligned with the `:`? (See https://github.com/google/TensorNetwork/blob/be652dde0c1f3f147bbf04b9999b6605717c6b22/tensornetwork/network.py#L768) for an example)

Reply:
Ignore what I said (except for the formatting).